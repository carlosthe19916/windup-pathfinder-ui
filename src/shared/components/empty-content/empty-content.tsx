import React from "react";
import {
  EmptyState,
  EmptyStateIcon,
  Title,
  EmptyStateBody,
  EmptyStateVariant,
  Bullseye,
} from "@patternfly/react-core";
import { BatteryEmptyIcon } from "@patternfly/react-icons";

export const EmptyContent: React.FC = () => {
  return (
    <Bullseye>
      <EmptyState variant={EmptyStateVariant.small}>
        <EmptyStateIcon icon={BatteryEmptyIcon} />
        <Title headingLevel="h4" size="lg">
          No data to show
        </Title>
        <EmptyStateBody>
          There is nothing to show. Make sure you selected a Graph database to
          read from.
        </EmptyStateBody>
      </EmptyState>
    </Bullseye>
  );
};
