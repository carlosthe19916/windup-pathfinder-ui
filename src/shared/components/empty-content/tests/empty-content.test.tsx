import React from "react";
import { shallow } from "enzyme";
import { EmptyContent } from "../empty-content";

describe("EmptyContent", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(<EmptyContent />);
    expect(wrapper).toMatchSnapshot();
  });
});
