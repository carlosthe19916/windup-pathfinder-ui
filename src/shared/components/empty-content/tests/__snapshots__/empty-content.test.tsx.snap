// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyContent Renders without crashing 1`] = `
<Bullseye>
  <EmptyState
    variant="small"
  >
    <EmptyStateIcon
      icon={[Function]}
    />
    <Title
      headingLevel="h4"
      size="lg"
    >
      No data to show
    </Title>
    <EmptyStateBody>
      There is nothing to show. Make sure you selected a Graph database to read from.
    </EmptyStateBody>
  </EmptyState>
</Bullseye>
`;
